src/sd.jl

```diff
@@ -246,7 +246,7 @@ function reshape_set(::MOI.PositiveSemidefiniteConeSquare, ::SquareMatrixShape)
 end
 vectorize(matrix::Matrix, ::SquareMatrixShape) = vec(matrix)
 
-function vectorize(matrix, shape::Union{SymmetricMatrixShape,SquareMatrixShape})
+function vectorize(matrix, shape::SquareMatrixShape)
```

Comment:
I also wonder if it's necessary to call `Matrix(vec(matrix))`. (This method and the one above.) Can we get away with just `vec(matrix)`?

Reply:
If it's not a `Matrix` then it's going to build a `ReshapedArray`